A reflection on going static

steveoh opened the pull request:
Pull request description
------------------------

waiting on reviews

brigsz commented:
@steveoh I am confused on this one, I don't know what it means

brigsz commented:
okay, @steveoh I do not understand "We focused on GitHub as we thought that would create the most friction - we overlooked and simplified or ignored all of the other pain points."

brigsz commented:
okay, Generally after a few reads I can follow things, however this article is definitely aimed at someone familiar with website production.  I am unsure if that was your intended audience, if it was no worries, if it was not, then more explanation on "markdown" and other similar things (static/dynamic) are needed. I ask myself who will read this?  If it is the rank and file GIS person, then maybe we need to have a closing paragraph that loops it all together in non web development speak.

steveoh commented:
@brigsz 

> okay, @steveoh I do not understand "We focused on GitHub as we thought that would create the most friction - we overlooked and simplified or ignored all of the other pain points."

I would say that when we introduced the idea of the static website to the office, we focused heavily on github and how to use it. We didn't spend near as much time on the finer details that people got hung up on when creating and editing content. This was mainly because we didn't think about certain things or they were still in motion. Github was kind of like opening the door, but once everyone was mostly inside we kinda didn't have a good path to the fridge for that 🍺. I think we thought markdown was so simple we hit it briefly etc. 

>this article is definitely aimed at someone familiar with website production. 

yeah this article is kind of like a what to expect if you are thinking of doing something similar or just want to know what we experienced doing it.

> then more explanation on "markdown" and other similar things (static/dynamic) are needed.

I'll add links to the websites so people can learn more about them if interested.

> we need to have a closing paragraph that loops it all together in non web development speak.

Do you want to take a stab at that? I'm not too sure what i would say.

stdavis commented on _posts/2017-01-17-forced-git-hub-a-reflection-on-a-new-website-using-git-hub-and-jekyll.md:
```diff
@@ -0,0 +1,54 @@
+
+### Green Grass
+
+We are pleased with our website and hope you are too! We all rely on our [notes and cheatsheets](https://github.com/agrc/gis.utah.gov/blob/master/README.md) and our workflows for being successful. GitHub and git are hard. Markdown is difficult. Jekyll is complicated. We thought it would be fun and useful as an office to learn to use GitHub and the other technologies involved in a static website.  ...
```
>The forced learning process could have been smoother and if we did it again we would adjust our approach but we made it happen as a team.

The forced learning process could have been smoother. If we did it again we would adjust our approach. But in the end we made it happen as a team.
